lib/foreman_wreckingball/engine.rb

```diff
@@ -7,6 +7,14 @@ module ForemanWreckingball
   class Engine < ::Rails::Engine
     engine_name 'foreman_wreckingball'
 
+    WRECKINGBALL_STATUSES = [
```

Comment:
Why do you safe them as a string here? You usually call `.constantize` on them anyways.

Reply:
these classes are not loaded yet here, so when you replace string with class name you'll get `NameError: uninitialized constant ForemanWreckingball::ToolsStatus`